vignettes/getting-started-with-teal.Rmd

```diff
@@ -35,75 +34,87 @@ library(teal)
+</span>
+
+Every `teal` application is composed of the following elements, all of which can be controlled by the app developer by passing arguments to the `init` function:
+
+* <span style="color: #BF37F3;">Application Title</span> _(browser's tab title)_: is the title of the application.
+* <span style="color: #FB6251;">Application Header and Footer</span> _(the top and the bottom of the app)_: any content to be placed at the top and bottom of the application.
+* <span style="color: #f2b73f;">Teal Modules</span> _(tabs under the header)_: tab for each module included in the application.
+  * In the example code: there are two modules named "Module 1" and "Module 2".
+* <span style="color: #83cc41;">Module Content</span> _(panel on the middle)_: the outputs of the currently active module.
+* <span style="color: #3A88FE;">Filter Panel</span> _(panel on the right hand side)_: for filtering the data to be passed into all `teal` modules.
+  * In the example code: the filter panel is being initialized with a filter for the `Species` variable in the `iris` dataset.
 
 ## Creating your own applications
 
-The key function to use to create your `teal` application is `init`, which requires two arguments: `data` and `modules`.
+The key function to use to create your `teal` application is `init`, which requires two mandatory arguments: `data` and `modules`. There are other optional arguments for `init`, which can be used to customize the application. Please refer to the documentation for `init` for further details.
 
 ### Application data
 
-The `data` argument to the `init` function specifies the data used by your application.
-In the example above this is data that exists in the global environment.
-We call `teal_data` to bind all datasets into one R object of class `teal_data`.
-We could also specify relationships between the datasets using the `join_keys` argument but in this case the datasets are not related.
+The `data` argument in the `init` function specifies the data used in your application. All datasets which are about to be used in `teal` application must be passed through `teal_data` object.
+It is also possible to specify relationships between the datasets using the `join_keys` argument but in this case the datasets are not related. See [this vignette](including-data-in-teal-applications.html) for details.
 If data is not available and has to be pulled from a remote source, `init` must receive a `teal_data_module` that specifies how to obtain the desired datasets and put them into a `teal_data` object.
 See [this vignette](data-as-shiny-module.html) for details.
 
-In order to use `CDISC` clinical trial data in a `teal` application the `cdisc_data` function is used instead.
+In order to use CDISC clinical trial data in a `teal` application the `cdisc_data` function is used instead. If you would like to use custom `SDTM` standards, you can use the `teal_data` along with `join_keys` to specify the relationships between the datasets.
```

Comment:
```suggestion
In order to use CDISC clinical trial data in a `teal` application the `cdisc_data` function is used instead. 
Other standards, like SDTM, can be handled by applying `teal_data` and `join_keys`.
```

Reply:
This is misleading. `SDTM` and CDISC are somewhat analogous, they could not be called other standards.
CDISC is the organization that sets the standards. SDTM is the standard set by this organization. CDISC also provides guidelines to create custom SDTM datasets which is why sponsors (pharma companies) can create their own SDTM datasets. I would prefer the previous document as is.